python/lsst/imgserv/imageREST_v0.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/python
+
+# LSST Data Management System
+# Copyright 2015 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+
+"""
+This module implements the RESTful interface for Image Cutout Service.
+Corresponding URI: /image
+
+@author  Jacek Becla, SLAC
+"""
+import os
+import uuid
+
+from flask import Blueprint, Flask, make_response, request
+
+from lsst.imgserv.locateImage import dbOpen
+
+imageREST = Blueprint('imageREST', __name__, template_folder='imgserv')
+
+# this will eventually print list of supported versions
+@imageREST.route('/')
+def index():
+    return """
+Hello, LSST Image Cutout Service here. Try something like:<br />
+/image/v0/raw?ra=1&dec=1&filter=r<br />
+/image/v0/raw/cutout?ra=1&dec=1&filter=r&width=12&height=12
+"""
+
+
+# this will handle something like:
+# GET /image/v0/raw?ra=1&dec=1&filter=r
+@imageREST.route('/raw', methods=['GET'])
+def getRaw():
+    ra = request.args.get('ra', '1')
+    dec = request.args.get('dec', '1')
+    filter = request.args.get('filter', 'r')
+
+    # fetch the image here
+    # TODO - check inputs are valid.
+    ra = float(ra)
+    dec = float(dec)
+    print "raw ra={} dec={} filter={}".format(ra, dec, filter)
+    # fetch the image here
+    w13Raw = dbOpen("~/.lsst/dbAuth-dbServ.txt")
+    imgFull = w13Raw.getImageFull(ra, dec)
+    print "Full w={} h={}".format(imgFull.getWidth(), imgFull.getHeight())
+    fileName = str(uuid.uuid4())
+    imgFull.writeFits(fileName)
```

Comment:
Will this write to a file in current directory? Should it use some temporary directory instead, current directory may not be writable.